test_sec_filings/sec_filings/test_section_api.py

```diff
@@ -276,6 +350,35 @@ def test_section_narrative_api_csv_response(form_type, response_type, section, t
     assert [x["text"] for x in response_list]
 
 
+@pytest.mark.parametrize(
+    "form_type, response_type, section",
+    [
+        ("10-K", "text/csv", "RISK_FACTORS"),
+        ("10-Q", "text/csv", "RISK_FACTORS"),
+        ("S-1", "text/csv", "RISK_FACTORS"),
+        ("10-K", "text/csv", "_ALL"),
+        ("10-Q", "text/csv", "_ALL"),
+        ("S-1", "text/csv", "_ALL"),
+    ],
+)
+def test_section_narrative_api_csv_response_with_unsupported_response_schema(form_type, response_type, section, tmpdir):
+    sample_document = generate_sample_document(form_type)
+    filename = os.path.join(tmpdir.dirname, "wilderness.xbrl")
+    with open(filename, "w") as f:
+        f.write(sample_document)
+
+    # NOTE(robinson) - Reset the rate limit to avoid 429s in tests
+    app.state.limiter.reset()
+    client = TestClient(app)
+    response = client.post(
+        SECTION_ROUTE,
+        files=[("text_files", (filename, open(filename, "rb"), "text/plain"))],
+        data={"output_format": response_type, "output_schema": "unsupported", "section": [section]},
+    )
+    assert response.status_code != 200
```

Comment:
let's assert the exact value on the `status_code` here. and, instead of printing below, let's assert the value of the json response.